projects/project-player-library/src/lib/pages/task-details-page/task-details-page.component.ts

```diff
@@ -0,0 +1,20 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+
+@Component({
+  selector: 'lib-taskdetails-page',
+  // standalone: true,
```

Comment:
Remove the comments

Reply:
its done the task details page was not pushed so the changes is not reflecting.